packages/components/pages/f-contact-preferences/src/services/mapping/_tests/contactPreferences.mapper.test.js

```diff
@@ -0,0 +1,53 @@
+import {
+    mapToPreferencesViewModel,
+    mapToPreferencesUpdateModel,
+    filterSortPreferences
+} from '../contactPreferences.mapper';
+
+import {
+    contactPreferencesGetResponse,
+    contactPreferencesViewModel,
+    contactPreferencesUpdateModel,
+    filteredContactPreferencesModel
+} from '../../../../test-utils/setup';
+
+describe('ContactPreferencesMapper', () => {
+    describe('When calling `mapToPreferencesViewModel`', () => {
+        it('should return the expected view model', () => {
+            // Arrange
+            const expected = contactPreferencesViewModel;
+
+            // Act
+            const actual = mapToPreferencesViewModel(contactPreferencesGetResponse.data);
+
+            // Assert
+            expect(actual).toEqual(expected);
+        });
+    });
+
+    describe('When calling `mapToPreferencesUpdateModel`', () => {
+        it('should return the expected update model', () => {
+            // Arrange
+            const expected = contactPreferencesUpdateModel;
+
+            // Act
+            const actual = mapToPreferencesUpdateModel(contactPreferencesViewModel);
+
+            // Assert
+            expect(actual).toEqual(expected);
+        });
+    });
+
+    describe('When calling `filterSortPreferences`', () => {
```

Comment:
Does this need to cover the return of the empty array when say `preferences` do not exist?